packages/lib/response/cache.ts

```diff
@@ -0,0 +1,50 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateResponses {
+  environmentId?: string;
+  personId?: string;
+  responseId?: string;
+  singleUseId?: string;
+  surveyId?: string;
+}
+
+export const responseCache = {
+  tag: {
+    byEnvironmentId(environmentId: string) {
+      return `environments-${environmentId}-responses`;
+    },
+    byResponseId(responseId: string) {
+      return `responses-${responseId}`;
+    },
+    byPersonId(personId: string) {
+      return `person-${personId}-responses`;
```

Comment:
shouldn't this be `people-${personId}-responses` (instead of person)

Reply:
Done